Description: Fix incompatible pointer types
GCC 14 no longer allows implicitly casting all pointer types to all other pointer types
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-01-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cool_mzx/cool_mzx.c
+++ b/cool_mzx/cool_mzx.c
@@ -239,7 +239,7 @@
 	}
 }
 
-Uint32 timeri(Uint32 interval)
+Uint32 timeri(Uint32 interval, void *param)
 {
 	if(!laite)
 	{
